src/renderers/dom/stack/client/ReactDOMComponent.js

```diff
@@ -583,6 +584,20 @@ ReactDOMComponent.Mixin = {
         );
         didWarnShadyDOM = true;
       }
+
+      if (__DEV__) {
+        if (this._namespaceURI === DOMNamespaces.html) {
+          warning(
+            !(document.createElement(this._tag) instanceof window.HTMLUnknownElement) ||
```

Comment:
Well, we cant really test SVG elements using `HTMLUnknownElement`. Thus i will use `ownerDocument` as you suggested